code/src/test/java/com/google/sps/storage/BusinessTest.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.storage;
+
+import org.junit.Test;
+import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+ 
+@RunWith(JUnit4.class)
+public final class BusinessTest {
+
+  @Test
+  public void createBusiness() {
+    Business business = new Business("Test");
+    Assert.assertFalse(business.contactedInLastWeek());
```

Comment:
nit: you can have a static import of assertTrue and assertFalse. That tends to be the preferred strategy for such things:

import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertTrue;

Reply:
Why is static required for the import?